src/scripts/main.js

```diff
@@ -358,3 +358,40 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+
+document.addEventListener('DOMContentLoaded', function () {
+  // Знаходимо таблицю з класом dashboard
+  const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the HTML contains a table element with the class 'dashboard'. If it does not exist, the script will not append any rows, and this could lead to confusion. Consider adding a check to ensure the table exists before attempting to append rows.